.github/workflows/pull-request.yml

```
@@ -39,8 +39,14 @@ jobs:
         run: |
           chmod +x ./.github/workflows/scripts/countline.py
           ./.github/workflows/scripts/countline.py --lines 600 --exclude_files src/screens/LoginPage/LoginPage.tsx src/GraphQl/Queries/Queries.ts src/screens/OrgList/OrgList.tsx src/GraphQl/Mutations/mutations.ts src/components/EventListCard/EventListCardModals.tsx src/components/TagActions/TagActionsMocks.ts src/utils/interfaces.ts src/screens/MemberDetail/MemberDetail.tsx
-
-      - name: Get changed TypeScript files
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)